diff --git a/node_modules/@rollup/plugin-commonjs/dist/index.js b/node_modules/@rollup/plugin-commonjs/dist/index.js
index 6024f3b..989e94e 100644
--- a/node_modules/@rollup/plugin-commonjs/dist/index.js
+++ b/node_modules/@rollup/plugin-commonjs/dist/index.js
@@ -106,20 +106,20 @@ function equals(a, b, strict) {
 }
 
 const PROXY_SUFFIX = '?commonjs-proxy';
-const getProxyId = id => `\0${id}${PROXY_SUFFIX}`;
+const getProxyId = id => '\0' + id + PROXY_SUFFIX;
 const getIdFromProxyId = proxyId => proxyId.slice(1, -PROXY_SUFFIX.length);
 const EXTERNAL_SUFFIX = '?commonjs-external';
-const getExternalProxyId = id => `\0${id}${EXTERNAL_SUFFIX}`;
+const getExternalProxyId = id => '\0' + id + EXTERNAL_SUFFIX;
 const getIdFromExternalProxyId = proxyId => proxyId.slice(1, -EXTERNAL_SUFFIX.length);
 const VIRTUAL_PATH_BASE = '/$$rollup_base$$';
 const getVirtualPathForDynamicRequirePath = (path, commonDir) => {
   if (path.startsWith(commonDir)) return VIRTUAL_PATH_BASE + path.slice(commonDir.length);
   return path;
 };
-const DYNAMIC_REGISTER_PREFIX = '\0commonjs-dynamic-register:';
-const DYNAMIC_JSON_PREFIX = '\0commonjs-dynamic-json:';
-const DYNAMIC_PACKAGES_ID = '\0commonjs-dynamic-packages';
-const HELPERS_ID = '\0commonjsHelpers.js'; // `x['default']` is used instead of `x.default` for backward compatibility with ES3 browsers.
+const DYNAMIC_REGISTER_PREFIX = '\0' + 'commonjs-dynamic-register:';
+const DYNAMIC_JSON_PREFIX = '\0' + 'commonjs-dynamic-json:';
+const DYNAMIC_PACKAGES_ID = '\0' + 'commonjs-dynamic-packages';
+const HELPERS_ID = '\0' + 'commonjsHelpers.js'; // `x['default']` is used instead of `x.default` for backward compatibility with ES3 browsers.
 // Minifiers like uglify will usually transpile it back if compatibility with ES3 is not enabled.
 
 const HELPERS = `
