diff --git a/node_modules/servor/servor.js b/node_modules/servor/servor.js
index 39734d7..9c2096e 100644
--- a/node_modules/servor/servor.js
+++ b/node_modules/servor/servor.js
@@ -122,7 +122,7 @@ module.exports = async ({
   // Notify livereload clients on file change
 
   reload &&
-    fs.watch(root, { recursive: true }, () => {
+    fs.watch(root + '/dist', { recursive: true }, () => {
       while (clients.length > 0)
         sendMessage(clients.pop(), 'message', 'reload');
     });
